tests/unit/disco/sagas/testDiscovery.js

```diff
@@ -0,0 +1,104 @@
+import SagaTester from 'redux-saga-tester';
+
+import { loadEntities } from 'core/actions';
+import { ErrorHandler } from 'core/errorHandler';
+import addonsReducer from 'core/reducers/addons';
+import apiReducer from 'core/reducers/api';
+import * as api from 'disco/api';
+import { getDiscoResults, loadDiscoResults } from 'disco/actions';
+import discoResultsReducer from 'disco/reducers/discoResults';
+import discoverySaga from 'disco/sagas/discovery';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import {
+  createFetchDiscoveryResult, fakeDiscoAddon,
+} from 'tests/unit/disco/helpers';
+
+
+describe('disco/sagas/discovery', () => {
```

Comment:
oh, I wish it were that simple but it's not. There are many situations (because JavaScript) where Jest will give us no indication *whatsoever* which test file is producing an error. Here is an example:

<img width="666" alt="screenshot 2017-08-11 10 15 36" src="https://user-images.githubusercontent.com/55398/29219606-20b884da-7e7e-11e7-8606-9d2138cb382d.png">

If that spec wasn't named `disco/sagas/discovery` how would you figure out where the error is coming from? You'd have to start grepping for phrases in the test description which is cumbersome because we have a lot of overlapping test descriptions. 

I see your point about renaming files and forgetting to update specs but until we have a better solution I'd like to put the test file path in all specs. I've definitely struggled to locate failing tests that have vague spec names before and I want to save my future self from that pain.